src/internal.c

```diff
@@ -18782,24 +18782,32 @@ int ProcessReplyEx(WOLFSSL* ssl, int allowSocketErr)
                                        &ssl->curRL, &ssl->curSize);
 
 #ifdef WOLFSSL_DTLS
-            if (ssl->options.dtls &&
-                    (ret == SEQUENCE_ERROR || ret == DTLS_CID_ERROR)) {
-                WOLFSSL_MSG("Silently dropping DTLS message");
-                ssl->options.processReply = doProcessInit;
-                ssl->buffers.inputBuffer.length = 0;
-                ssl->buffers.inputBuffer.idx = 0;
+            if (ssl->options.dtls) {
+                if ((ssl->options.handShakeDone && !IsEncryptionOn(ssl, 0))
```

Comment:
For DTLSv1.3 it would be nice to not allow epoch 0 messages after {client_hello/server_hello}, but before the handshake is done as well

Reply:
This is a fix targeted for 1.2. I will address the 1.3 specific aspect later.